### PR TITLE
Done validations

### PR DESCRIPTION
**Pull request by mwaurajr**
All is done.

### CODE REVIEW

File: app/models/example.rb [hunk 1]
```diff
@@ -1,2 +1,3 @@
 class Example < ApplicationRecord
+    validates :description, presence: true
```
[the remark this answers is not in the document]
Comment (mwaurajr) [reply app/models/example.rb:2]: Thought you said example 😔😞, is alright lemme work on it.

**Comment by mwaurajr** [comment]
Oooh no I forgot an end. OMG I had done this so I just did it fast coz I knew what was needed, only that the PR were not visible on your side. It's ok I will redo it. Thank you. Just troubles of a junior dev.

File: app/models/example.rb (hunk 1 above)
[the remark this answers is not in the document]
Comment (mwaurajr) [reply app/models/example.rb]: I have done the corrections to this error. Also, try running it on the server there are some syntax errors on the form based on the routes. I've tested validations on the console and they're working well.

**Comment by gathuku** [comment]
> Oooh no I forgot an end. OMG I had done this so I just did it fast coz I knew what was needed, only that the PR were not visible on your side. It's ok I will redo it. Thank you. Just troubles of a junior dev.

@mwaurajr No problem. I wanted to know the too you're using so you can use to ruby 3.1.2  instead of switching to 2.7

**Comment by mwaurajr** [comment]
I am using rbenv.

**Comment by mwaurajr** [comment]
Screenshot from 2023-06-13 18-02-34
error

**Comment by gathuku** [comment]
@mwaurajr this could be because `@product` variable is nil. Are you getting the error with the current changes or steps to reproduce.

**Comment by mwaurajr** [comment]
Yes I think so I've taken time on the set_product medthod and yes it maybe because product is just nill

**Comment by mwaurajr** [comment]
Haven't used discord. How about I text you my number on twitter.

**Review by gathuku — APPROVED** [review]

**Comment by gathuku** [comment]
Nice work @mwaurajr

**Pull request merged**